cmd/libs/go2idl/conversion-gen/generators/conversion.go

```diff
@@ -122,6 +133,16 @@ func existingConversionFunctions(context *generator.Context) conversions {
 	return preexisting
 }
 
+func isAPIPackage(path string) bool {
+	apiPackages := []string{"k8s.io/kubernetes/pkg/api/", "k8s.io/kubernetes/pkg/apis/"}
```

Comment:
like I said, I think you can just make an exception for "k8s.io/kubernetes/pkg/api/", just like the existing code.

I'm not sure if the go2idl Package object has an expanded path or not? We could consider adding it.

I can also imagine detecting these by looking for a declaration of a `type kubernetesAPITypePackage struct{}`? Or a special import? Or a comment somewhere? These would require code changes, but more explicit might be better.